develop/devguide/Connector/AdvancedDcfBestPractices.md

```diff
@@ -0,0 +1,65 @@
+
+## Best practices for interface tables within a connector
+
+A table should be as static as possible and implemented in a way that is similar to DVEs:
+
+- [RTDisplay](xref:Protocol.Params.Param.Display.RTDisplay) must be set to *true*.
+- A state column should be used to track the lifespan of an interface.
+- Logic should be present to remove a "Deleted" interface:
+  - Automatically with a toggle button.
+  - Manually with delete button in the row.
+  - With a button to delete all rows with deleted status.
+
+  > [!TIP]
+  > See: [Data Handling](xref:Data_handling)
+
+Tables generating DCF interfaces should not be updated or changed often, as this can result in an increased load on SLElement, SLProtocol and SLNet. Additionally, QActions that trigger on a change of a table generating DCF interfaces is not advised because this can cause deadlocks. DCF interfaces are created asynchronously so the QAction triggered may not immediately see the update.
+
+For large SNMP tables, it is recommended to that you use a custom table and copy the information needed to the custom table. This allows for full control over the table rows and information. Larger tables will have a greater impact on DMA performance. Filtering via bandwidth or otherwise is advised (consult the TAM and/or customer on what the best option is).
+
+> [!NOTE]
+> From DataMiner 10.1.5 onwards (RN29326), the [Group@isInternal](xref:Protocol.ParameterGroups.Group-isInternal) attribute can be used to specify that a DCF interface is internal. These interfaces will not be shown to the user and will only be available internally, limiting what is shown.
+
+### Interface alarm monitoring
+
+A pitfall for DCF interfaces is alarm monitoring. Every update to a table will initiate a recalculation for every interface created by that table. This means that for large tables with frequent updates (e.g. bit rates), there will be a lot of DCF alarm updates, which will consume a lot of resources.
+
+You can minimize or avoid this impact but will lose the functionality of seeing the interface alarm color linked to the alarm thresholds defined on the original table. To avoid this, you can either use the custom table method (described above) or make use of the [Group@calculateAlarmState](xref:Protocol.ParameterGroups.Group-calculateAlarmState) attribute to disable the interface state calculation.
+
+## Best practices for creating connections
+
+### Internal
+
+Use the DCF Helper, which contains methods for you to create connections safely using SLNet. Make sure these connections are only updated or created as necessary. Additional updates will increase the load on the SLProtocol and SLNet processes.
+
+### External
+
+External connections can be managed in several ways:
+
+- **Manager Element**: Creates connections between elements given information on the connections.
+- **Automation script**: No library for Automation scripts, but it is possible using SLNet via EditConnection message.
```

Comment:
Could you rephrase line 50 and 51? Right now, these aren't entirely clear to me.